src/App.tsx

```diff
@@ -1,24 +1,349 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Error } from './components/Error';
+import { Todo } from './types/Todo';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { Filters } from './types/Filters';
 import { UserWarning } from './UserWarning';
+import { ErrorMessage } from './types/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 11689;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [filter, setFilter] = useState(Filters.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [pageIsLoaded, setPageIsLoaded] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isPageLoading, setIsPageLoading] = useState(false);
```